src/controllers/petsController.js

```diff
@@ -144,10 +145,24 @@ const patchPet = async (req, res, next) => {
     });
 };
 
+const getBreeds = async (req, res, next) => {
```

Comment:
Do we need `postBreeds` if a user want to add a breed that is not in the database?

Reply:
I currently don't plan for that... We can have an "Other" breed for each type of pets